config/200-clusterrole.yaml

```diff
@@ -60,6 +60,7 @@ metadata:
   labels:
     serving.knative.dev/release: devel
     serving.knative.dev/controller: "true"
+    networking.knative.dev/provider: istio
```

Comment:
Any reason to choose networking.knative.dev and not keep this as serving.knative.dev/networkProvider?

Reply:
No particular reason.  I renamed.